client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -49,7 +49,35 @@ class ScanPage extends Component {
 	}
 
 	renderThreats() {
-		return <p>Threats found.</p>;
+		const { threats, site } = this.props;
+
+		return (
+			<>
+				<h1 className="scan__header">{ translate( 'Your site may be at risk' ) }</h1>
+				<p>
+					{ translate(
+						'The scan found {{strong}}%(threatCount)s{{/strong}} potential threats with {{strong}}%(siteName)s{{/strong}}.',
+						{
+							args: {
+								siteName: site.name,
+								threatCount: numberFormat( threats.length ),
+							},
+							components: { strong: <strong /> },
+							comment:
+								'%(threatCount)s represents the number of threats currently identified on the site, and $(siteName)s is the name of the site.',
```

Comment:
What is the purpose of the comment? I read the `i18n-calypso` README and I didn't find any good explanation of it.

Reply:
@rcanepa the translators read it for context. It's hard to consistently write good comments but I try to imagine that I'm a translator translating string after string with minimal context and what I would want to know in that situation. So if the string isn't clear out of context, or if there are variables like this, or anything like that then it is a good idea to leave a comment.

Reply:
Oh, I didn't know about the translators. Good to know!